conans/assets/templates/new_v2_autotools.py

```diff
@@ -38,6 +37,7 @@ def layout(self):
 
         def generate(self):
             at_toolchain = AutotoolsToolchain(self)
+            at_toolchain.configure_args = at_toolchain.shared_configure_args()
```

Comment:
Don't love this UI, seems a bit spaguetti, as everything is inside ``AutotoolsToolchain``, it seems like something like ``at_toolchain.add_shared_flags()`` would be cleaner?